site/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Enabling Shared Array Buffer
+authors:
+  - jakearchibald
+description: >
+  Browsers are imposing new requirements on SharedArrayBuffer usage. Learn how to enable it cross-browser and cross-platform.
+origin_trial:
+  url: /origintrials/#/view_trial/303992974847508481
+date: 2021-01-11
+hero: image/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/VwqAOKnWHDx3nJegh94L.jpg
+alt: Iframes, images, and windows being accessed by someone 'evil'
+---
```

Comment:
We may want to add tags for this article. I put together a guide here which explains how to add tags: https://github.com/GoogleChrome/developer.chrome.com/pull/219

Because every tag generates its own page we usually try to be pretty coarse with our tags (e.g. preferring "security" as opposed to "shared-array-buffer"). This is mostly so we don't end up with a lot of pages that only have a single post.

Reply:
With web.dev, I usually cargo-culted the frontmatter from another article, but obviously couldn't do that here 😄 .

Is there anything else I might be missing?

Reply:
> Is there anything else I might be missing?

I just pushed a commit to this PR because I noticed the image on the blog card looked a bit off.

I gave the image on the cards an explicit height but noticed that it looked a bit blurry on my non-retina desktop monitor. I think it's because [the image is really wide](https://developer-chrome-com.imgix.net/image/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/VwqAOKnWHDx3nJegh94L.jpg?auto=format).

What do you think about cropping the image and uploading another version?

Unfortunately the homepage card is a tad larger than the blog card so the image aspect ratios are a little different but they're in the same ballpark:

414 x 208 (homepage)
328 x 184 (blog landing page)

Reply:
I took that image from another web.dev article, so I don't have the original. I can try and find a new image. What is the recommended upload size?

Reply:
at its largest the image will be displayed at 960 x 480. So I guess 1920 x 960 to support 2x displays. As an image expert I'm curious what your thoughts are? We could go even higher if you think we should be supporting higher resolutions.

Reply:
I think supporting 2x is enough for an image that's usually photographic